src/main/java/frc/robot/constants/fields/CompetitionField.java

```diff
@@ -0,0 +1,18 @@
+package frc.robot.constants.fields;
+
+import edu.wpi.first.wpilibj.geometry.Translation2d;
+import frc.robot.constants.FieldConstants;
+
+/**
+ * Competition field way-points. relative to the (0,0) point in PathWeaver. all
+ * measurement are measured in meters.
+ */
+public class CompetitionField extends FieldConstants {
+    // TODO: check reality
+    public CompetitionField() {
+        TRENCH_RUN_ENTRANCE = new Translation2d(5.45, -0.70485);
```

Comment:
constants conventions are now k...
update `TRENCH_RUN_ENTRANCE` to `kTrenchRunEntrance`
do the same for other constants